create_daily_notion_page.py

```
@@ -44,24 +52,151 @@ def build_children(tasks):
+    """Get all child page blocks of a parent."""
+    url = f"{BLOCKS_URL}/{parent_id}/children"
+    all_pages = []
+    has_more = True
+    start_cursor = None
+    
+    while has_more:
+        params = {"page_size": 100}
+        if start_cursor:
+            params["start_cursor"] = start_cursor
+            
+        response = requests.get(url, headers=HEADERS, params=params)
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)